po/fr.po

```diff
@@ -3556,7 +3556,7 @@ msgstr ""
 
 #: /tmp/fish/explicit/share/functions/cdh.fish:5
 msgid "Select directory by letter or number: "
-msgstr "Sélectionner un dossier par lettre ou chiffre"
+msgstr "Sélectionner un dossier par lettre ou chiffre : "
```

Comment:
This now also has a space *before*. Are you sure that's correct?

Reply:
In french, we always write spaces after *and* before `:`. For us it's kinda special to see `!?:;` without beeing spaced, unlike in our strange language.

So yes it's intentionnal.